Fix Text Class's bugs in shaders branch (the same as #1030)

Tony031218 opened the pull request:
Fixed Three bugs of Text Class.
This pull request is the same as #1030
Without this, Text Class wouldn't work in the 'shaders' branch.

NavpreetDevpuri commented:
i also changed text_mobject.py in https://github.com/3b1b/manim/pull/1018 
i also update file to newer version but in file changes it shows that i added also lines that you added, but its should show only my changes. does it create any problem or its okay ?

Tony031218 commented:
> i also changed text_mobject.py in #1018
> i also update file to newer version but in file changes it shows that i also added lines that you added, but its should show only my changes. does it create any problem or its okay ?

This pull request is of shaders branch not master. If you want update text_mobject.py you can just pull the newer version from master, because my pull request #1030 has already been merged.

NavpreetDevpuri commented:
thanks 
understood!
